CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Fixed false positive results from the `export` command that were caused
+  by the ARCHIVE component returning data unrelated to the issued query.
+
```

Comment:
Thanks for adding the entry. However, it doesn't fit the style in which we write the changelog. If you look at the other bug entries, we always describe the bug first, with a complete sentence. Then we describe the fix. For the user, the issue was duplicate results. Since we're writing this for the user, I think this aspect should be the center of the message. Here's a suggestion:
```
- 🐞 Some queries returned duplicate events because the archive did not filter
  the result set properly. This no longer occurs after fixing the table slice
  filtering logic.
```